initial_config.yaml

```
@@ -0,0 +1,95 @@
+data_config:
```

YAMLlint (1.35.1)

[error] wrong new line character: expected \n

(new-lines)